### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by kukamesud**

### CODE REVIEW

**Comment by driskull** [comment]
Hey @kukamesud, It looks like these changes are just for the language files. Was this PR intentional? 

I'm not seeing anything with the upload parts.

**Comment by kukamesud** [comment]
Hi Matt,

I wanted to create a new folder in 'js/nls/' and translate the
'resource.js' file for albanian language.
If you could please advice me for the necessary steps.

Thanks in advance

File: sq/resources.js [hunk 1]
```diff
@@ -0,0 +1,319 @@
+define({
+  root: ({
```
Comment (driskull) [inline sq/resources.js]: remove the `root: ({`

File: sq/resources.js [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+      btnLoadMoreText: "Load More",
+      unavailableTitleText: "Untitled",
+      unavailableConfigMessage: "Unable to load config",
+      share: "Share",
+      viewReportsTabText: "Reports",
+      viewLegendTabText: "Legend",
+      viewAboutusTabText: "About",
+      appLoadingFailedMessage: "There was an error loading the viewer"
+    }
+  }),
```
Comment (driskull) [inline sq/resources.js]: remove `}),`

File: sq/resources.js [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+  "ar": 1,
```
Comment (driskull) [inline sq/resources.js]: remove lines 204-318

**Comment by driskull** [comment]
thanks!

**Comment by driskull** [comment]
This was added as well: https://github.com/Esri/geoform-template-js/commit/ad4514f2125aeff146308cf2d425f8595e452303

**Comment by kukamesud** [comment]
Hi Matt,

Thanks for adding the language files.
Could you please tell me how can I configure the GeoForm so when I publish
it on ArcGIS Online it appears in Albanian language to the audience?

Thanks in advance

**Comment by driskull** [comment]
Hey @kukamesud, no problem.

To get this working on ArcGIS Online, you will have to...

1. Host this application on your own server
2. Create a web mapping application item in your organization and point it to the URL where you're hosting this app.
3. Share the application item in a group
4. Add that group to your organization's applications in your org settings
5. Share a webmap using the application you created.

More details can be found here: http://doc.arcgis.com/en/arcgis-online/create-maps/create-app-templates.htm#ESRI_SECTION1_9F809A4A343D4245A3C72F65687CD8AD

-Matt

**Comment by kukamesud** [comment]
Thanks Matt

**Pull request merged**